src/index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <!-- linthtml-configure attr-bans=false -->
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="page">
+    <div class=
+      "calendar calendar--start-day-wed calendar--month-length-30">
```

Comment:
You should count each classname as a separate attribute if you have more than 2 of them
So in this case better to make each of them into a separate line
```suggestion
    <div class="
      calendar 
      calendar--start-day-wed 
      calendar--month-length-30
    ">
```